tests/test_cli.py

```diff
@@ -59,11 +59,11 @@ def test_schema_convert_cli(self, runner, config_path, helpers):
     # get manifest by default
     # by default this should download the manifest as a CSV file
     @pytest.mark.google_credentials_needed
-    def test_get_example_manifest_default(self, runner, helpers, config, data_model_jsonld):
+    def test_get_example_manifest_default(self, runner, helpers, config: Configuration, data_model_jsonld):
         output_path = helpers.get_data_path("example.Patient.manifest.csv")
 
         result = runner.invoke(
-            manifest, ["--config", config.CONFIG_PATH, "get",  "--data_type", "Patient", "--jsonld", data_model_jsonld]
+            manifest, ["--config", config.config_path, "get",  "--data_type", "Patient", "--jsonld", data_model_jsonld]
```

Comment:
tests here are not working because config.config_path returns None

Reply:
@andrewelamb I am also getting this error. I find myself unable to provide the path to get around the error: ```
esult = runner.invoke(manifest, ["--config","/Users/mialydefelice/Documents/schematic_e/schematic/tests/data/test_configs/valid_config.yml","get",  "--data_type", "Patient", "--jsonld", data_model_jsonld])
(Pdb) result
<Result FileNotFoundError(2, 'No such file or directory')>
```
So perhaps something needs to be changed to allow these tests to pass.

Reply:
@andrewelamb this test failing

Reply:
@mialy-defelice But as shown it's not due to the configuration class, but something with the test itself.

Reply:
Well its passing on `develop` but failing on this branch, so it warrants further investigation.

Reply:
That's fair, I'll play around with it tomorrow and see if I can figure out the issue.

Reply:
@linglp What file does `<Result FileNotFoundError(2, 'No such file or directory')>` refer to?

Reply:
OK I figured it out, its two different files that are causing `<Result FileNotFoundError(2, 'No such file or directory')>`

As is `config.CONFIG_PATH is None` since by default there is no config file which causes the FileNotFoundError

When Mialy does
 `result = runner.invoke(manifest, ["--config","/Users/mialydefelice/Documents/schematic_e/schematic/tests/data/test_configs/valid_config.yml","get",  "--data_type", "Patient", "--jsonld", data_model_jsonld])`

whats happens is `schematic.utils.google_api_utils.pybuild_service_account_creds` tries to get the service acount creds. But since the config she is using is in the thest configs folder, the configuration class is returning a path for the service account creds in the same directory `Users/mialydefelice/Documents/schematic_e/schematic/tests/data/test_configs/schematic_service_account_creds.json`
Which aren't there causing the `FileNotFoundError`.

To get this to work you can either move your service account credentials to that folder.
Or can load the example config which is at the top level were the service account credentials should be:

```
    @pytest.mark.google_credentials_needed
    def test_get_example_manifest_default(self, runner, helpers, config: Configuration, data_model_jsonld):
        output_path = helpers.get_data_path("example.Patient.manifest.csv")
        config.load_config("config_example.yml")

        result = runner.invoke(
            manifest, ["--config", config.config_path, "get",  "--data_type", "Patient", "--jsonld", data_model_jsonld]
        )


        assert result.exit_code == 0
        self.assert_expected_file(result, output_path)
```
The above works for me, does this work for everyone?

Reply:
@andrewelamb yes this works for me now, can you push those changes?

Reply:
@mialy-defelice Just pushed those

Reply:
@linglp Can your approve this?